trulens/visualizations.py

```diff
@@ -999,3 +1012,301 @@ def __call__(
             attrs_input, x, output_file, blur, threshold, masked_opacity,
             combine_channels
         )
+
+
+class Output(ABC):
```

Comment:
nit:TextOutput
This looks like it deals specifically with text

Reply:
Don't mean it to handle text only but that is what is available presently.